package.json

```diff
@@ -4,8 +4,8 @@
   "description": "Make company global travel and accommodation easy and convenient for the strong workforce of savvy members of staff, by leveraging the modern web",
   "main": "index.js",
   "scripts": {
-    "build": "npx babel src --out-dir dist",
     "start": "npm run build && node ./dist/index.js",
+    "build": "npx babel src --out-dir dist",
```

Comment:
would you mind adding  `--copy-files` at the end of your build script so you won't have issues with swagger? but if u don't have that issue then you can procced

Reply:
@okafor-chidimma Do you need to do this?

Reply:
Okay...Let me do that